src/pages/dmt5/_trade-control.js

```diff
@@ -0,0 +1,178 @@
+import { graphql, useStaticQuery } from 'gatsby'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Flex, SectionContainer } from 'components/containers'
+import { Header, QueryImage, Text } from 'components/elements'
+import { localize, Localize } from 'components/localization'
+import device from 'themes/device'
+
+const query = graphql`
+    query {
+        swap_calculator: file(relativePath: { eq: "swap-calculator.png" }) {
+            ...fadeIn
+        }
+        margin_calculator: file(relativePath: { eq: "margin-calculator.png" }) {
+            ...fadeIn
+        }
+    }
+`
+const Section = styled(SectionContainer)`
+    display: flex;
+    padding: 0 12rem 0 0;
```

Comment:
padding-right?

Reply:
Since `SectionContainer ` has default padding, I wanted to make it `0`.

Reply:
i see, youre right